app/components/candidate_interface/course_choices_review_visa_status_component.html.erb

```diff
@@ -1,18 +1,20 @@
 <% if can_sponsor_visa? %>
   <%= title %>
 <% else %>
-  <%= govuk_inset_text(classes: 'app-inset-text--narrow-border app-inset-text--important') do %>
+  <%= govuk_inset_text(classes: 'app-inset-text--narrow-border app-inset-text--important govuk-!-margin-top-0 govuk-!-padding-bottom-1') do %>
     <p class="govuk-heading-s app-inset-text__title"><%= title %></p>
 
     <p class="govuk-body">
-      You said that you do not yet have the right to work or study in the UK.
+      Check your eligibility for different <%= govuk_link_to('visa types and immigration statuses', t('train_to_teach_in_england.url')) %>.
```

Comment:
Can we link to the place in the guidance which helps the candidate with this? e.g. here: https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#visa